content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -166,15 +148,21 @@ Note that [`NoAction`](#noaction) can be used in replace of [`Restrict`](#restri
 | PostgreSQL | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
 | MySQL      | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**†    |
 | SQLite     | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
-| SQLServer  | **✔️**  | **❌**   | **✔️**   | **✔️**  | **✔️**     |
+| SQL Server | **✔️**  | **❌**   | **✔️**   | **✔️**  | **✔️**     |
 
 - † On [InnoDB](https://dev.mysql.com/doc/refman/5.6/en/innodb-introduction.html#:~:text=InnoDB%20is%20a%20general%2Dpurpose,clause%20creates%20an%20InnoDB%20table.), `SetDefault` will be ignored and silently use `NoAction`.
 
+From version 3.6.0, referential actions will also be available in MongoDB if the `[referentialIntegrity](/concepts/components/prisma-schema/relations/referential-integrity)` Preview feature is enabled:
```

Comment:
Does this render correctly? I think I would have put the backticks _inside_ the link.

Reply:
Thanks, fixed